content/documentation/pages/1-installation/3-kubernetes/3.kubectl.md

```diff
@@ -201,6 +203,118 @@ grafana-info:
   url: 'https://grafana:3000'
 ```
 
+#### Wavefront
+
+Metrics for the Spring Cloud Data Flow server along with deployed streams and tasks can be sent to the Wavefront service.
+Before enabling Wavefront ensure you have your Wavefront URL and API token.
+
+First create a secret which will encode your API token in a file called `wavefront-secret.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: wavefront-api
+  labels:
+    app: wavefront
+data:
+  wavefront-api-token: bXl0b2tlbg==
+```
+
+The value of `wavefront-api-token` is a base64 encoded string representing your API token. For more information on Secrets, see the [Kubernetes Documentation](https://kubernetes.io/docs/concepts/configuration/secret/).
+
+Create the secret:
+
+```
+kubectl create -f wavefront-secret.yaml
+```
+
+To mount the secret and make it available to Spring Cloud Data Flow, modify the file `src/kubernetes/server/server-deployment.yaml` making the following additions:
+
+```yaml
+spec:
+  containers:
+    - name: scdf-server
+      volumeMounts:
+        - name: wavefront-api
+          mountPath: /etc/secrets/wavefront-api
```

Comment:
@chrisjs I realized that we need a note saying this mount path needs to belong to whatever `SPRING_CLOUD_KUBERNETES_SECRETS_PATHS` is configured to. Isn't it?

Reply:
i don't think its needed as one can see it already but if you have some wording you would like to add im open to it

Reply:
I thought the `mountPath` has to be in the paths that `SPRING_CLOUD_KUBERNETES_SECRETS_PATHS` is configured. I could only make it to work when I modified the `mountPath` to something that `SPRING_CLOUD_KUBERNETES_SECRETS_PATHS` has it.

Reply:
in `server-deployment.yaml` we have:

```
        - name: SPRING_CLOUD_KUBERNETES_SECRETS_PATHS
          value: /etc/secrets
```

?